python/mhs/import_scenarios.py

```diff
@@ -232,10 +300,13 @@ def main():
     verbose_message("Reading CSV files {0} and {1}\n".format(
         site_file, gmf_file))
 
+    from earthquake_scenarios import read_event_set
```

Comment:
There's a reason why this import is not a top level?

Reply:
Yes, but not a very good reason.  Moved - earthquake_scenarios still wants to be replaced...